src/payment/strategies/paypal-commerce/paypal-commerce-script-loader.spec.ts

```diff
@@ -26,18 +26,53 @@ describe('PaypalCommerceScriptLoader', () => {
         paypalLoader = new PaypalCommerceScriptLoader(loader);
     });
 
-    it('loads PayPalCommerce script  with client Id and currency USD', async () => {
-        const output = await paypalLoader.loadPaypalCommerce({ clientId: 'aaa', currency: 'USD' });
-
-        expect(loader.loadScript).toHaveBeenCalledWith('https://www.paypal.com/sdk/js?client-id=aaa&currency=USD', {async: true, attributes: {}});
-        expect(output).toEqual(paypal);
+    function checkParamsInScript(options: PaypalCommerceScriptOptions, equalStrings: string[]) {
```

Comment:
Generally speaking, we avoid putting expectations inside a function for the sake of getting the tests DRY. This is because the abstraction makes the tests harder to follow. So I think it's okay for you to duplicate some code in your tests, as long as you can make the expectations clearer.

Reply:
Okay. I understand. Thank you!